From 25fac269993dbed904b9e106033c7ed2d12b2c1a Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Mon, 14 Nov 2022 13:57:41 +0200
Subject: [PATCH] [SW-0] ppboot: greco: arm: increase sram size limitation

Increase limitation for ppboot sram footprint when building for ARM,
because the PLL fix introduced in the following patch is causing the
sram footprint to exceed the current limit.
This has no effect on the spi flash footprint limit.
ppboot Arm is not used regularly on greco and kept as a fallback option

Change-Id: I2e761f44e60eaaa98488219977aa60156c1d3b1d
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/greco/arm/common.h b/arch/greco/arm/common.h
index d95a272..fb99d33 100644
--- a/arch/greco/arm/common.h
+++ b/arch/greco/arm/common.h
@@ -10,7 +10,7 @@
 #include <bitops.h>
 #include <sizes.h>
 
-#define PPBOOT_SIZE			0x5000		/* 20KB */
+#define PPBOOT_SIZE			0x6000		/* 24KB */
 #define PREBOOT_SIZE			0x14000		/* 80KB */
 
 #define HL_SP_SRAM_SIZE			SZ_64K
